python/task_manager/background_task_manager.py

```diff
@@ -355,7 +337,7 @@ def _get_worker_thread(self):
         # create a new worker thread - note, there are two different implementations of the WorkerThread class
         # that use two different recipes.  Although WorkerThreadB is arguably more correct it has some issues
```

Comment:
Can this stuff about workerThreadB be deleted? It feels like it refers to some outdated code now.